crates/backend/src/codegen.rs

```diff
@@ -281,12 +281,12 @@ impl ToTokens for ast::Struct {
                     let ptr = #wasm_bindgen::convert::IntoWasmAbi::into_abi(value);
 
                     #[link(wasm_import_module = "__wbindgen_placeholder__")]
-                    #[cfg(all(target_arch = "wasm32", any(target_os = "unknown", target_os = "none")))]
+                    #[cfg(all(target_arch = "wasm32", any(target_os = "unknown", target_os = "none", target_os = "emscripten")))]
```

Comment:
I'd say we should change all these checks to just `target_arch = "wasm32"`, no `target_os`.

Excluding Emscripten used to be the primary reason those were added, and we want to experiment with WASI too, so it doesn't make sense to have an allowlist that lists every possible Wasm OS at this point.